File: jput-core/src/main/java/cz/encircled/jput/io/TrendResultReader.java
package cz.encircled.jput.io;

import cz.encircled.jput.model.PerformanceTestRun;

/**
 * @author Vlad on 21-May-17.
 */
public interface TrendResultReader {

    long[] getStandardSampleRuns(PerformanceTestRun newRun, int standardSampleSize);

}

File: jput-core/src/main/java/cz/encircled/jput/io/TrendResultReader.kt
package cz.encircled.jput.io

import cz.encircled.jput.model.PerformanceTestExecution

/**
 * @author Vlad on 21-May-17.
 */
interface TrendResultReader {

    fun getReferenceExecutions(newExecution: PerformanceTestExecution, referenceExecutionsCount: Int): List<Long>

}
